mapc_rhbp_ettlinger/StockItem[]     stock_items